tests/test_predicates.py

```diff
@@ -137,3 +144,6 @@ def test_lift_matrix(self):
         self.assertTrue(equal(target, lift_matrix(base, list_test)))
         self.assertTrue(equal(target, lift_matrix(base, vector_test)))
         self.assertTrue(equal(target, lift_matrix(base, matrix_test)))
+    '''
```

Comment:
We should delete this. Take anything good from it and make it actual tests then delete it cause comments this long of just code are ridiculous.